data/mods/Magiclysm/enchantments/Sun_Mage.json

```diff
@@ -21,7 +21,41 @@
     "type": "enchantment",
     "id": "SUN_MAGE",
     "condition": "ALWAYS",
-    "values": [ { "value": "PERCEPTION", "add": 5 } ],
     "hit_me_effect": [ { "id": "solar_blinding" } ]
+  },
+  {
+    "type": "effect_on_condition",
```

Comment:
Might wanna use a `deactivate_condition ` here. See EFFECT_ON_CONDITION.md for documentation.

Reply:
Also, would it be possible for the sun mage trait to cancel out albino and sun sensitive traits? If so, could be nice to have.

Reply:
> deactivate_condition 

I dont know, id prefer to have a better way to start this EoC, so it wont work all time, checking "does the player get an attunement", than deactivate it when... when what? when the player will lost the trait (which is impossible without debug mode)?

> albino and sun sensitive

are hardcoded :(

Reply:
If you have a deactivate_condition for not: has trait SUN_MAGE, then the EOC should get deactivated at the start of the game. Then each time the player gets a trait all deactivated EOC's are run again. If the gained trait was sun mage, then this EOC would become active again until you lose the sun mage trait. At least that's my understanding of it.

Reply:
if thats so, then yes, it would be good to do (because i thought the game checks "does player has an attunement" like each hour from the start of the game till the very end)